lib/purescript-cst/src/Language/PureScript/CST/Errors.hs

```diff
@@ -63,6 +63,7 @@ data ParserWarningType
   | WarnDeprecatedConstraintInForeignImportSyntax
   | WarnDeprecatedKindImportSyntax
   | WarnDeprecatedKindExportSyntax
+  | WarnDeprecatedCaseOfOffsideSyntax
```

Comment:
I had no idea what to call this error, so this is what I put for now. Let me know if you have a better idea for it.